releasenotes/notes/remove-deprecated-py-env-vars-c4a67902c6aaa10b.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Remove unused environment variables `DD_AGENT_PY` and `DD_AGENT_PY_ENV` from known environment variables in flare command.
```

Comment:
I'm not sure this warrants customers' attention in the release notes -- would anyone need to know this occurred?

Reply:
Hm I wasn't sure whether to include it or not... I was thinking it could be nice to give a heads up in case anyone happened to still be setting these or was going to search for usages.

Reply:
I don't mind removing it though if you think it's not useful!

Reply:
Ah, that makes sense -- I was thinking that these variables had been removed in another PR in this release, which would have mentioned them.  But I was incorrect! Sorry!